src/member-api.js

```diff
@@ -46,24 +47,45 @@ export class MemberApi extends TypedEmitter {
     this.#dataTypes = dataTypes
   }
 
+  /** @type {Array<any>}
+   * @property {import('./capabilities.js').RoleId} 0
+   * @property {import('./capabilities.js').Capability} 1 */
+
   /**
    * @param {string} deviceId
    *
    * @param {Object} opts
-   * @param {import('./capabilities.js').RoleId} opts.roleId
+   * @param {import('./generated/rpc.js').Invite_RoleName} opts.roleName
+   * @param {string} [opts.roleDescription]
```

Comment:
Not sure the params should change here. To me, the roleId should still be used here, and the name and description sent with the invite are derived based on the capabilities associated with the roleId.

The other (tangential) uncertainty i have is how this all works with translations. if the name and description are built into the capability definition, how does internationalization work when you want to surface those things are part of the UI? (easier for name than description in that case)

thoughts @gmaclennan ?

Reply:
I think what you say makes sense, but syncing this morning with gregor we arrived at that change maybe there's some reason I'm forgetting??

Reply:
hm i see. if this was discussed already then feel free to disregard my initial comment! i admit the advantage of providing the description is that there's more freedom from the consumer to describe the role/capability, as opposed to us hardcoding it in the capability definition

Reply:
Yes good points @achou11, it was late last night when discussing and I hadn't fully wrapped my head around this - I was wondering where the frontend would get the roleId from, but having a fixed role name doesn't allow the frontend to internationalize what is sent in the invite.

1. We need to export the DEFAULT_CAPABILITIES from @mapeo/core so the front end can read those - ideally they are accessible from the mapeo client without needing to do an async call to the backend - maybe an opportunity to use package.exports @achou11 :) 
2. We keep `roleId` as the required param here, and both role name and role description optional. Role name can default to the `name` on the capability record. The frontend can pass a translated role name (the invitor would decide on the translation, rather than the invitee - because in the future capabilities and capability names might not be available on a device before it joins a project)
3. Need to add a check for a valid role ID and throw if invalid.

Reply:
If we keep `roleId`, shouldn't we remove `roleName` from params since it can be grabbed from the `roleId`. I dunno about `roleDescription` since `DEFAULT_CAPABILITIES` doesn't have a `roleDescription` (maybe the idea is to add it to that record?)

Reply:
> If we keep roleId, shouldn't we remove roleName from params since it can be grabbed from the roleId.

it's still useful for roleName to be an optional param (not a required param) because otherwise we wouldn't be able to easily provide a translated string. Gregor's suggestion is to have the optional name param and if that isn't provided use the name associated with the roleId. For example, an application should be able to do `members.invite({ roleName: "Miembro", ... })` if they want the invited device to see the translated name of the role. Otherwise the default is the english name we're using in the role id definition